web/server/codechecker_server/auth/cc_ldap.py

```diff
@@ -303,10 +303,11 @@ def get_ldap_query_scope(scope_form_config):
     # the default scope is the subtree
     if scope_form_config == 'base':
         return ldap.LDAP_SCOPE_BASE
-    elif scope_form_config == 'one':
+
+    if scope_form_config == 'one':
```

Comment:
Elif could be better solution here also.

Reply:
Same answer.